### PR TITLE
chore(starters): add bunch of starters

### PR DESCRIPTION
**Pull request by smakosh**
## Description
Added my personal starters:
- Airtable starter
- Apollo starter
- Rest API starter with Auth
- Portfolio one page starter

### CODE REVIEW

**Comment by endormi** [comment]
Well 😅

**Comment by freiksenet** [comment]
@smakosh We can merge it once you fix the tags, see Danger result.

**Comment by smakosh** [comment]
Yup, we need to add the new tags I added on this PR

**Comment by smakosh** [comment]
All looks good now @endormi @freiksenet

File: docs/categories.yml [hunk 1]
```diff
@@ -54,6 +54,13 @@ starter:
   - Styling:None
   - Testing
   - Transitions
+  - Apollo
```
Comment (freiksenet) [inline docs/categories.yml]: Please don't add new tags.

Comment (smakosh) [reply docs/categories.yml]: it was mentioned on Gatsby's docs: https://www.gatsbyjs.org/contributing/submit-to-starter-library/#adding-new-tag

Comment (freiksenet) [reply docs/categories.yml]: While we support adding tags, we strongly discourage that. We have had plenty of tags that only have 1 instance and that made the UI confusing. We've just made a cleanup to remove them. Therefore the correct solution for your starters would be not adding new tags, but removing tags that don't exist from your entries.

Thank you!

**Comment by freiksenet** [comment]
Thank you very much @smakosh!

**Pull request merged**